modules/README.md

```diff
@@ -9,6 +9,8 @@ Modules should be entirely self-contained into a subdirectory with the following
 
         css/          - Any necessary new CSS classes/files should go here
 
+        help/         - Should contain a markdown file if this module is using a module class
```

Comment:
I would just say something like "Contains a markdown file of help content for each page in this module" since there should always be a module class.